README.md

```diff
@@ -69,11 +62,7 @@ repositories {
   ...
   maven {
     name = "GithubPackages"
-    url = uri("https://maven.pkg.github.com/sf-recco/android-sdk")
-    credentials {
-      username = extra["gprUser"].toString()
-      password = extra["gprKey"].toString()
-    }
+    url = uri("https://recco-maiden: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@maven.pkg.github.com/sf-recco/android-sdk")
```

Comment:
out of curiosity, where is that `github_pat_11....` created from?

Reply:
from this lady: https://github.com/recco-maiden xD

Reply:
but this approach won't work, the PAT was detected by Github during the push commit and disabled it. so this approach is not feasible, sadly